tests/integration/backup_tests/test_backups.py

```diff
@@ -467,13 +464,9 @@ async def test_restore_new_cluster(ops_test: OpsTest, continuous_writes_to_db, c
             writes_in_new_cluster == writes_in_old_cluster
```

Comment:
A few lines above (line 458) - There is a reference to a non-existing function: `ha_helpers.count_writes(ops_test)`